README.md

```diff
@@ -132,68 +142,69 @@ Run the following command to apply this configMap configuration to the cluster:
 kubectl apply -f Application/manifests/container-azm-ms-agentconfig.yaml
 ```
 
-## Pod Annotations for Scraping
+### Pod Annotations for Scraping
```

Comment:
I like the call out for teaching purposes. It might get missed and if someone wants to implement from scratch, it's important to know about the annotations.